tools/build_velox.sh

```diff
@@ -49,7 +49,7 @@ if [ $BUILD_VELOX_FROM_SOURCE == "ON" ]; then
     echo "VELOX_INSTALL_DIR=${VELOX_INSTALL_DIR}"
     mkdir -p $VELOX_SOURCE_DIR
     mkdir -p $VELOX_INSTALL_DIR
-    git clone https://github.com/oap-project/velox.git -b main $VELOX_SOURCE_DIR
+    git clone https://github.com/rui-mo/velox.git -b wip_config $VELOX_SOURCE_DIR
```

Comment:
commit to oap-project/velox?

Reply:
Yes, there is a PR for that, and this change is just for testing and will be reverted before merge. The gluten PR and velox PR will be merged together.